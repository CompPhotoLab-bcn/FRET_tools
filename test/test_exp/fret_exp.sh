../../src/fret_exp.py  naproxen_295nm.in naproxen_295nm.pdf --x_title "Emission wavelength (nm)" --y_title "Intensity" --plot_title "naproxen (excitation at 295 nm)" --width 7.0 --height 3.5 --wavelength 315 --show_plot True
