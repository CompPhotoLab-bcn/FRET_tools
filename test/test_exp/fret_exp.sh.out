                                                                              
                                                                  ▓▒▒▒▒       
                                                      ░▓▓▒▒▒    ▒▓▒▒▒▒░       
                                ░▓▒▒▒▒    ▓▓▒▒░░░    ▓▒▒░░░░░   ░░▒▒▒  ▒      
           ░▒░░░░░    ▒▒░░░░    ▒▒░░░░░   ░░▒░░░░░   ░░░▒░░░░░ ░░░░░     ▒    
             ░░░░    ░░░░░░░   ░░░▒░░░░  ░░░░▒░░░░░ ░░░░░▒░░░░░▒░░░           
              ░░░░░  ░░░▒░░░░░ ░░░░▒░░░░░░░░░ ▒▒░░▒░▒░░░  ▓▒▒▒▓▓▓▒▒           
              ▒▒▒▒▒░░░░░ ▒▒▒▒▒▒░░░░ ▓▒▒▒▒▒░░░  ▒▒▒▒▓▓▒▒░                      
           ░    ▓▓▓▓▓▒▒░   ▓▓▓▓▓▒▒                                            
   ░     ____                      ____  _           _        _          _    
       / ___|___  _ __ ___  _ __ |  _ \| |__   ___ | |_ ___ | |    __ _| |__  
      | |   / _ \| '_ ` _ \| '_ \| |_) | '_ \ / _ \| __/ _ \| |   / _` | '_ \ 
 ░    | |__| (_) | | | | | | |_) |  __/| | | | (_) | || (_) | |__| (_| | |_) |
       \____\___/|_| |_| |_| .__/|_|   |_| |_|\___/ \__\___/|_____\__,_|_.__/ 
   ░                       |_|                                                
    ░                                                        ▓▓▒▒▒▒▒          
      ▒░░░ ▒                                        ░▓▒▒▒  ░▒▒▒▒░   ░         
          ░░░░░                ▓▒▒     ▓▓▓▒▒▒    ▒▓▓▒░░░░░ ░░░░░     ▒ ▒      
          ░░░░     ▒▒▒▒▒▒    ▒▒▒░░░    ▒▒▒░░░░   ░░░░▒░░   ░░  ░      ▒▒      
          ░░░░    ░░░░░░░░  ░░▒░░░░   ░░░░░░░░  ░░░░░░ ░░░░░░░░               
          ▒▒▒░░ ░░░░░░░  ░  ░░░▒▒░░░  ░░░░░░░░░░░░░░    ▒▒▒▒▒▒▓               
           ░▓▓▒▒░░░ ░░░░░░ ░░░  ▒▒▒▒▒▒▒░░░ ▒▒▒▒▒▒▒▒░░                         
                ▓░░  ▒░░░░░░░░     ▒▓▓▓░    ░▓▓▓▒                             
                     ▓▒▒▒▒▒▒░                                                 
                                   ╭-------------------------------------╮    
                                   |     F R E T - E X P vers 1.0        |    
                                   ╞ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ═ ╡    
                                   | Carles Curutchet                    |    
                                   | Ozge Ergun                          |    
                                   | www.ub.edu/cplab                    |    
                                   | www.allodd-itn.eu                   |    
                                   ╰-------------------------------------╯    

Reference spectra (Protein donor only):
Ligand concentration (acceptor): 0.0 μM
Protein concentration (donor): 5.0 μM
Wavelength: 315.0 nm
Reference intensity at 315.0 nm 2.7

Calculation of binding constant: Linear regression data:
log([L]): 1.0000, log(F0/F - 1): -0.7154
log([L]): 1.6990, log(F0/F - 1): -0.2115
log([L]): 2.0000, log(F0/F - 1): -0.0457

Equation to fit:
log(F0/F - 1) = log(Kb) + n log([L])

Linear regression results:
Slope (n, binding molecularity): 0.6787
Intercept (log(Kb)): -1.3873
Association constant Kb: 0.0410 μM⁻¹
Dissociation constant Kd: 24.3922 μM
Correlation coefficient (r): 0.9983

FRET efficiency uncorrected/corrected by percent of free protein vs complex:
   l_concentration  p_concentration  ...  percent_complex  efficiency_corrected
1             10.0              5.0  ...         0.290763              0.555362
2             50.0              5.0  ...         0.672113              0.566265
3            100.0              5.0  ...         0.803909              0.589237

[3 rows x 8 columns]
